modules/highgui/src/cap_dshow.cpp

```diff
@@ -2246,7 +2246,7 @@ int videoInput::getVideoPropertyFromCV(int cv_property){
         case CV_CAP_PROP_GAMMA:
             return VideoProcAmp_Gamma;
 
-        case CV_CAP_PROP_MONOCROME:
+        case CV_CAP_PROP_MONOCHROME:
```

Comment:
But then again, we will need double cases everywhere. @vpisarev is this really necessary?